sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -39,25 +41,21 @@
 from ._sync_token import SyncTokenPolicy
 from ._user_agent import USER_AGENT
 
-if TYPE_CHECKING:
-    from azure.core.paging import ItemPaged
-    from azure.core.credentials import TokenCredential
-
 
 class AzureAppConfigurationClient: # pylint: disable=client-accepts-api-version-keyword
-    # pylint:disable=line-too-long
     """Represents a client that calls restful API of Azure App Configuration service.
 
     :param str base_url: base url of the service
     :param credential: An object which can provide secrets for the app configuration service
-    :type credential: :class:`~azure.appconfiguration.AppConfigConnectionStringCredential` or :class:`~azure.core.credentials.TokenCredential`
+    :type credential: :class:`~azure.appconfiguration.AppConfigConnectionStringCredential`
+     or :class:`~azure.core.credentials.TokenCredential`
```

Comment:
nit: I would indent the second line to align with what other docstrings do
```suggestion
    :type credential: :class:`~azure.appconfiguration.AppConfigConnectionStringCredential`
        or :class:`~azure.core.credentials.TokenCredential`
```